interface/orders/orders_results.php

```diff
@@ -650,7 +650,7 @@ function validate(f) {
             "form_result_abnormal[$lino]",
             'proc_res_abnormal',
             $result_abnormal,
-            xl('Indicates abnormality'),
+            xlt('Indicates abnormality'),
```

Comment:
Note that should only by htmlescaping stuff right before it is sent to html output. Should not escape above since this is a function call (which will do it's own html escaping on the html output). So should revert above line.